CSharp/DevVmPowershell/Helpers/Implementations/ImportApiHelper.cs

```diff
@@ -129,7 +129,8 @@ static void ImportJobOnComplete(JobReport jobReport)
 
 		static async Task<int> GetNumberOfDocumentsAsync(int workspaceId, string fileType)
 		{
-			HttpClient httpClient = RestHelper.GetHttpClient(InstanceAddress, AdminUsername, AdminPassword);
+			RestHelper restHelper = new RestHelper();

```

Comment:
- [ ] Pass RestHelper to the ImportApiHelper constructor and use it for making it DI

Reply:
@calimetikcura The method this is in is a static method. So the RestHelper can not be passed into the constructor and used. Should I make the method non static?

Reply:
ah. that's fine for now as you have not touched that method.